abcclassroom/example-data/sample_config.yml

```diff
@@ -0,0 +1,56 @@
+
+# The GitHub organization to host your GitHub classroom
+# Create a new organization on GitHub and change this to the name you gave it
+organization: earth-analytics-edu
+
+# (Optional) Short name for organization. If set, this gets used to prepend
+# the repositories instead of the full organization name, just to make
+# names shorter and less unwieldy
+organization-shortname: ea
+
+# Name of your course. Will be used to prefix repositories inside the
+# organization. The repositories will be named:
+# https://github.com/<organization>/<courseName>-template
+# https://github.com/<organization>/<courseName>-<studentName>
+# or, if organization-shortname is set:
+# https://github.com/<organization-shortname>/<courseName>-template
+# https://github.com/<organization-shortname>/<courseName>-<studentName>
+course_name: earth-analytics-bootcamp
```

Comment:
I like the idea of this quickstart function updating just this item in the config. so it can take the course name. and that course name is used to create the course_dir folder name and updates the config file. because there are so many other items in the config i think that might be the best way to go to get started. just one argument - course name is what i'm proposing and the function should check that the name has dashes in between words (hyphens) rather than spaces